Maple2.Model/Game/User/Home.cs

```diff
@@ -26,6 +26,11 @@ public class Home : IByteSerializable {
     public string? Passcode { get; set; }
     public readonly IDictionary<HomePermission, HomePermissionSetting> Permissions;
 
+    public long DecorationExp { get; set; }
+    public long DecorationLevel { get; set; }
+    public long DecorationRewardTimestamp { get; set; }
+    public List<int> InteriorRewardsClaimed { get; set; }
```

Comment:
These should also be saved in the DB, yeah?

Reply:
yeah but its not even being handled yet :x